internal/datastore/spanner/readwrite.go

```diff
@@ -40,12 +39,6 @@ func (rwt spannerReadWriteTXN) WriteRelationships(ctx context.Context, mutations
 		}
 	}
 
-	if !rwt.disableStats {
```

Comment:
maybe we want to log a warning in the datastore initialization indicating the flag is basically ignored with spanner

Reply:
Meh, I don't think its necessary